com.unity.multiplayer.mlapi/Runtime/Core/NetworkBehaviour.cs

```diff
@@ -42,6 +42,9 @@ public enum __NExec
             Client = 2
         }
 
+        static private bool m_UseClassicDelta = true;
```

Comment:
Maybe add a comment that this will be removed later when we switch to snapshots? Or is the plan to keep this in and give the developer access?

Reply:
will do.